src/java.base/share/classes/java/util/Dictionary.java

```diff
@@ -39,6 +39,9 @@
  * <strong>NOTE: This class is obsolete.  New implementations should
  * implement the Map interface, rather than extending this class.</strong>
  *
+ * @param <K> the type of keys
+ * @param <V> the type of mapped values
+ *
```

Comment:
Urgh. This class is obsolete, but it was retrofitted to implement Map and was subsequently generified, so I'd update these to match java.util.Map.

Reply:
The javadoc of Dictionary states "The Dictionary class [...] maps keys to values." which was my guide for the wording, but I don't mind changing it.

Reply:
My bad, `Dictionary` was not retrofitted to implement `Map` but it gained `K` and `V` type parameters to align with `Map`. No need to change this; it doesn't really matter.